xds/internal/xdsclient/transport/loadreport.go

```diff
@@ -120,19 +120,19 @@ func (t *Transport) lrsRunner(ctx context.Context) {
 			defer cancel()
 			stream, err := v3lrsgrpc.NewLoadReportingServiceClient(t.cc).StreamLoadStats(streamCtx)
 			if err != nil {
-				t.logger.Warningf("Failed to create LRS stream: %v", err)
+				t.logger.Warningf("Creating LRS stream to server %q: %v", t.serverURI, err)
```

Comment:
I don't really like this style of not saying something bad happened, but printing an error string after the attempted operation instead.  Is this the new style?  E.g. `WARNING: Going swimming: water is cold` doesn't actually tell me if this means things broke completely, so it's not going swimming, or if it might still swim but perform in a degraded fashion.

Reply:
I conflated the concept of not having filler words in error strings (which tend to get repeated when errors are propagated up the stack) with log messages. I have changed everything in here to say that something actually failed. Thanks.